src/components/CornerCircularLabel/CornerCircularLabel.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './CornerCircularLabel.scss';
+import {Corner} from '../../services/CommonTypes'
+
+interface Props {
+    value: string | number,
+    corner: Corner
+}
+
+const CornerCircularLabel: React.FC<Props> = (props: Props) => {
+    const resolveCornerClass = () => {
+        switch (props.corner) {
+            case Corner.BOTTOM_RIGHT:
+                return 'bottom-right'
+            case Corner.BOTTOM_LEFT:
+                return 'bottom-left'
+            case Corner.TOP_LEFT:
+                return 'top-left'
+            case Corner.TOP_RIGHT:
+                return 'top-right'
+            default:
+                return ''
+        }
+    }
+    return (
+            <div className={"corner-circular-label " + (resolveCornerClass())} onClick={(e) => e.stopPropagation()}>
```

Comment:
Do you have a good reason to stop further event propagation here?

Reply:
Yup. The label is later on attached to a `CircularButton` and I don't want the button click to be triggered when the label is clicked. I see the point that for the label to be more general component this woudn't work but in my particular case it solves the problem. Probably could find a better design tough